app/templates/calendar/add_week.html

```diff
@@ -1,21 +0,0 @@
-{% for week in weeks_block %}
-<div class="week">
-    {% for day in week.days %}
```

Comment:
Why have you removed `add_week.html`, `calendar.html` and `layout.html`?

Reply:
because it was old files that ode said need to be deleted.their now in diffrent directory